.github/workflows/bench.yml

```diff
@@ -58,9 +61,7 @@ jobs:
 
       # Disable turboboost, hyperthreading and use performance governor.
       - name: Prepare machine
-        run: |
-          sudo /root/bin/prep.sh
-          echo "PERF_CMD=record -o perf.data -F997 --call-graph lbr -g" >> "$GITHUB_ENV"
+        run: sudo /root/bin/prep.sh
 
       # Pin the benchmark run to core 0 and run all benchmarks at elevated priority.
       - name: Run cargo bench
```

Comment:
https://github.com/mozilla/neqo/actions/runs/7959004190/job/21725490360?pr=1663#step:10:24 shows that nice isn't doing anything.

Reply:
Yes, it's on my list to check. I did give the bench user rights in limits.conf.